src/components/tasks/card/card.module.scss

```diff
@@ -5,6 +5,13 @@ $yellow: #ecef08;
 $white: #e5e5e5;
 $black: #252525;
 
+//*****utitlites****//
+.cursorPointer {
```

Comment:
Do we really need this `className` ?

Reply:
I have added it seperately to show the cursor as pointer only in edit mode

Reply:
Oh, then the className could be better for readability, I guess. Then `isEditMode` for classname would be better naming convention.
What are your thoughts on this since it shows `boolean` property ?

Reply:
LGTM  naming convention will make it easier to read and understand.

Reply:
I have made the change as rajat suggested, will close the convo's related to classnames. please confirm